build/ArtifactBuilder/CoreAgentComponents.cs

```diff
@@ -111,16 +130,26 @@ protected override void CreateAgentComponents()
 
             AgentApiDll = $@"{SourcePath}\..\_build\AnyCPU-{Configuration}\NewRelic.Api.Agent\netstandard2.0\NewRelic.Api.Agent.dll";
 
-            LinuxProfiler = Platform == "x64"
-                ? $@"{HomeRootPath}\newrelichome_x64_coreclr_linux\libNewRelicProfiler.so"
-                : null;
-
-            GRPCExtensionsLibLinux = new[]
+            LinuxProfiler = null;
+            GRPCExtensionsLibLinux = new string[0];
+            if (Platform == "x64") 
             {
-                $@"{HomeRootPath}\newrelichome_x64_coreclr_linux\libgrpc_csharp_ext.x64.so"
-            };
-
-            AgentInfoJson = $@"{SourcePath}\..\src\Agent\Miscellaneous\{Platform}\agentinfo.json";
+                LinuxProfiler = $@"{HomeRootPath}\newrelichome_x64_coreclr_linux\libNewRelicProfiler.so";
+                GRPCExtensionsLibLinux = new[]
+                {
+                    $@"{HomeRootPath}\newrelichome_x64_coreclr_linux\libgrpc_csharp_ext.x64.so"
+                };
+            } 
+            else if (Platform == "arm64") 
+            {
+                LinuxProfiler = $@"{HomeRootPath}\newrelichome_arm64_coreclr_linux\libNewRelicProfiler.so";
+                GRPCExtensionsLibLinux = new[]
+                {
+                    $@"{HomeRootPath}\newrelichome_arm64_coreclr_linux\libgrpc_csharp_ext.arm64.so"
+                };
+            }
+
+            AgentInfoJson = Platform == "arm64" ? null : $@"{SourcePath}\..\src\Agent\Miscellaneous\{Platform}\agentinfo.json";
```

Comment:
Does having a `null` here cause a problem? It looks like it gets put in a HashSet, which I know is allowed, but not always expected.

Reply:
This is kind of uncharted territory, since the HashSet isn't actually used in any code for the arm64 platform. I'll change the impl to not add any potential nulls though so this doesn't break something in the future